docker/python-base/install_conda.sh

```diff
@@ -1,22 +1,24 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-wget -q https://repo.anaconda.com/miniconda/Miniconda3-py38_4.9.2-Linux-x86_64.sh -O ~/miniconda.sh
-/bin/bash ~/miniconda.sh -b -p /opt/conda
-rm ~/miniconda.sh
-ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
-/opt/conda/bin/conda update -n base -c defaults conda
-/opt/conda/bin/conda init bash
+curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh
```

Comment:
[Miniforge](https://github.com/conda-forge/miniforge) makes it harder to mess up conda channel preference.

[mamba](https://github.com/mamba-org/mamba) has crucial tooling for dependency fixes, like `mamba repoquery whoneeds`.